server/service/confluence_service.go

```diff
@@ -0,0 +1,198 @@
+package service
```

Comment:
Later on we should make a client abstraction of the Confluence API if there's not one around yet.

Reply:
Sure, logged an [issue](https://github.com/mattermost-community/mattermost-plugin-confluence/issues/130) for the same.